packages/dashboard-frontend/src/components/ImportFromGit/RepoOptionsAccordion/GitRepoOptions/index.tsx

```diff
@@ -87,16 +87,13 @@ export class GitRepoOptions extends React.PureComponent<Props, State> {
     this.props.onChange(gitBranch, remotes, devfilePath, isValid);
   }
   public render() {
-    const { hasSupportedGitService } = this.props;
     const { gitBranch, remotes, devfilePath } = this.state;
     return (
       <Form isHorizontal={true} onSubmit={e => e.preventDefault()}>
-        {hasSupportedGitService && (
```

Comment:
We need to return logic for checking supported providers. Because, in the case of HTTP location, it will try to recognize the provider. If it is a supported provider, the branch can be changed. Otherwise, errors will occur when trying to set the branch.


Therefore, in the case of a private provider, we cannot recognize a supported one.